.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    allow:
+      dependency-type: "direct"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "diesel/reviewer"
```

```suggestion
      - "diesel/reviewers"
```
question: Should this be diesel/reviewers? 

Looks good to me other than this question! 👍